Assets/AnkrSDK/Scripts/Mobile/AnkrNetworkHelper.cs

```diff
@@ -0,0 +1,45 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using AnkrSDK.Core.Infrastructure;
+using AnkrSDK.Data;
+using Newtonsoft.Json;
+using UnityEngine;
+
+namespace AnkrSDK.Utils
+{
+	public class AnkrNetworkHelper : INetworkHelper
+	{
+		private static readonly string _networkQueryParameter = "network";
+		private static readonly string _logoQueryParameter = "logo";
+		private static readonly string _deepLinkURL = "https://metamask.app.link/dapp";
+		private static readonly string _switchNetworkPageLink = "switch-page.com";
+
+		public Task AddAndSwitchNetwork(EthereumNetwork network)
+		{
+			var url = GetURLFromNetwork(network);
+			AddAndSwitchCustomNetwork(url);
+			
+			return Task.CompletedTask;
+		}
+
+		private static void AddAndSwitchCustomNetwork(string url)
+		{
+			Debug.Log(url);
+//			Application.OpenURL(url);
```

Comment:
meaningless comments are not allowed, also this method does nothing, but it should, right?